lib/phlex/context.rb

```diff
@@ -76,7 +76,17 @@ def _attributes(attributes)
 
       buffer = first_render ? buffer = +"" : buffer = @_target
 
+      attributes.keys.each do |key|
+        if key.match /[<>&"']/
```

Comment:
I benchmarked `match` against `in` and `match` is faster. I also benchmarked joining the keys first and matching on the joined String with varying numbers of symbols. Joining first much slower.

Reply:
Looks like `match?` can be used here, which should be the fastest: https://github.com/fastruby/fast-ruby#regexp-vs-stringmatch-vs-string-vs-stringmatch-code-

Reply:
Thanks for this.